### PR TITLE
Enhancing env type built-in variable documentation

### PR DESCRIPTION
**Pull request by gacerioni**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [X] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [X] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/12_Variables-and-Expressions/harness-variables.md [hunk 1]
```diff
@@ -984,6 +984,17 @@ The description of the environment.
 
 The environment type, such as production or non-production.
 
+The available values are:
+- PreProduction
+- Production
```
Comment (michaelcretzman) [inline docs/platform/12_Variables-and-Expressions/harness-variables.md:989]:
```suggestion
- `PreProduction`
- `Production`
```

File: docs/platform/12_Variables-and-Expressions/harness-variables.md [hunk 2]
```diff
@@ -984,6 +984,17 @@ The description of the environment.
 
 The environment type, such as production or non-production.
```
Comment (michaelcretzman) [inline docs/platform/12_Variables-and-Expressions/harness-variables.md:985]:
```suggestion
The environment type, such as `Production` or `PreProduction`.
```

File: docs/platform/12_Variables-and-Expressions/harness-variables.md [hunk 3]
```diff
@@ -984,6 +984,17 @@ The description of the environment.
...
 
+The available values are:
+- PreProduction
+- Production
+
+You can evaluate it with JEXL expressions in the **Conditional Execution** Settings:
```
Comment (michaelcretzman) [inline docs/platform/12_Variables-and-Expressions/harness-variables.md:991]:
```suggestion
You can evaluate the expression using JEXL in the **Conditional Execution** settings of steps or stages:
```

**Review by michaelcretzman — CHANGES_REQUESTED** [review]
please incorp the suggestions and I will merge.

**Review by gabrielcerioni — APPROVED** [review]

**Review by michaelcretzman — APPROVED** [review]

**Comment by gabrielcerioni** [comment]
Thanks, Michael!

**Pull request merged**